src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,208 @@
+package duke.ui;
+
+import java.util.ArrayList;
+
+import duke.Storage;
+import duke.command.Command;
+import duke.exception.DukeException;
+import duke.task.Task;
+import duke.tool.Parser;
+import duke.tool.TaskList;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+/**
+ * Represents the UI manager for the system.
+ */
+public class Ui {
+    /**Welcome message*/
+    public static final String WELCOME_MESSAGE = "Hello, I am Duke! \n\t What can I do for you?";
+    /**
+     * Horizontal Line
+     */
+    private static final String LINE = "-------------------------------------";
+    private final Image user = new Image(this.getClass().getResourceAsStream("/images/me.png"));
+    private final Image duke = new Image(this.getClass().getResourceAsStream("/images/ben.png"));
+
+    /**
+     * Logo of Duke.
+     */
+    private final String logo = " ____        _        \n"
```

Comment:
Instead of spaces maybe we can use `\t`?

Reply:
Yes. I just never change it from the original repo. I think I can delete this part, since after using javaFX, this is useless.